src/metabase/analytics/prometheus.clj

```diff
@@ -198,7 +198,11 @@
   [(prometheus/counter :metabase-email/messages
                        {:description (trs "Number of emails sent.")})
    (prometheus/counter :metabase-email/message-errors
-                       {:description (trs "Number of errors when sending emails.")})])
+                       {:description (trs "Number of errors when sending emails.")})
+   (prometheus/counter :metabase-channel/http
+                       {:description (trs "Number of http calls made via channel.")})
```

Comment:
Maybe `Number of calls made via HTTP channel`? Because "via channel" sounds weird, we don't have some channel, it's always a concrete one, right?